src/index.html

```diff
@@ -12,5 +13,44 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form 
+      class="big__form"
+      action="#" 
+      method="post"
+      data-qa="big"
+      >
```

Comment:
```suggestion
    <form 
      class="big__form"
      action="#" 
      method="post"
      data-qa="big"
    >
```
Closing tag should be just under the opening
![image](https://user-images.githubusercontent.com/96409563/221942903-90e59349-0709-4e39-ad64-b9773d00f807.png)